Add rule docs for 0.16.1 integration updates (#1587)
elastic/security-docs: docs/detections/prebuilt-rules/downloadable-packages/0-16-1/prebuilt-rule-0-16-1-windows-firewall-disabled-via-powershell.asciidoc

[[prebuilt-rule-0-16-1-windows-firewall-disabled-via-powershell]]
=== Windows Firewall Disabled via PowerShell

> Identifies when the Windows Firewall is disabled using PowerShell cmdlets, which attackers do to evade network constraints, like internet and network lateral communication restrictions.

Reviewer: ```suggestion
Identifies when the Windows Firewall is disabled using PowerShell cmdlets, which can help attackers evade network constraints, like internet and network lateral communication restrictions.
```
